src/Symfony/Component/Stopwatch/StopwatchPeriod.php

```diff
@@ -25,20 +25,26 @@ class StopwatchPeriod
     /**
      * Constructor.
      *
-     * @param int $start The relative time of the start of the period (in milliseconds)
-     * @param int $end   The relative time of the end of the period (in milliseconds)
+     * @param int|float $start The relative time of the start of the period (in milliseconds)
+     * @param int|float $end   The relative time of the end of the period (in milliseconds)
      */
-    public function __construct($start, $end)
+    public function __construct($start, $end /*, $useMicroPrecision = false*/)
```

Comment:
No need to keep this commented, as it's still optional since you're adding a default value. So it won't break BC. (However you could add the `bool` typehint commented for 4.0)